src/ui-layout.js

```diff
@@ -796,6 +807,37 @@ angular.module('ui.layout', [])
             if(!element.hasClass('stretch')) element.addClass('stretch');
             if(!element.hasClass('ui-layout-container')) element.addClass('ui-layout-container');
 
+            scope.$watch('collapsed', function (val, old) {
+              if (old !== undefined && val !== old) {
```

Comment:
This is better as `if (angular.isDefined(old)) {` - `val` is guaranteed to not equal `old` by how $watch works.